core/test/old_names.py

```diff
@@ -0,0 +1,121 @@
+import netCDF4
+import numpy as np
+import numpy.ma as ma
+import time
+import math
+
+# Create a restart file that specifically uses the old dimension names
+if __name__ == "__main__":
+
+    # Grid dimensions. x varies fastest
+    nfirst = 1
+    nsecond = 1
+    nLayers = 1
+    ncg = 1
+    n_dg = 1
+    n_dgstress = 1
+    n_coords = 2
+    
+    
+    root = netCDF4.Dataset("old_names.nc", "w", format="NETCDF4")
+    
+    structure_name = "parametric_rectangular"
+    structgrp = root.createGroup("structure")
+    structgrp.type = structure_name
+    
+    metagrp = root.createGroup("metadata")
+    metagrp.type = structure_name
+    confgrp = metagrp.createGroup("configuration") # But add nothing to it
+    timegrp = metagrp.createGroup("time")
+    time_var = timegrp.createVariable("time", "i8")
+    data_time = 1263204000
+    time_var[:] = data_time
+    time.units = "seconds since 1970-01-01T00:00:00Z"
+    formatted = timegrp.createVariable("formatted", str)
+    formatted.format = "%Y-%m-%dT%H:%M:%SZ"
+    formatted[0] = "2010-01-01T00:00:00Z"
+    datagrp = root.createGroup("data")
+
+    nLay = datagrp.createDimension("z", nLayers)
+    yDim = datagrp.createDimension("y", nfirst)
+    xDim = datagrp.createDimension("x", nsecond)
+    yVertexDim = datagrp.createDimension("yvertex", nfirst + 1)
+    xVertexDim = datagrp.createDimension("xvertex", nsecond+ 1)
+    ycg_dim = datagrp.createDimension("y_cg", nfirst * ncg + 1)
+    xcg_dim = datagrp.createDimension("x_cg", nsecond * ncg + 1)
+    dg_comp = datagrp.createDimension("dg_comp", n_dg)
+    dgs_comp = datagrp.createDimension("dgstress_comp", n_dgstress)
+    n_coords_comp = datagrp.createDimension("ncoords", n_coords)
+    
+    field_dims = ("y", "x")
+    coord_dims = ("yvertex", "xvertex", "ncoords")
+    zfield_dims = ("z", "y", "x")
+
+    # Array coordinates
+    space = 25000. # 25 km in metres
+    node_x = np.zeros((nfirst + 1, nsecond + 1))
+    node_y = np.zeros((nfirst + 1, nsecond + 1))
```

Comment:
These don't seem to be used anywhere else.

Reply:
Indeed not, removed.